$MirOS: ports/www/lynx/patches/patch-src_LYUtils_c,v 1.3 2007/08/03 16:01:22 tg Exp $

	Important for upstream too

--- lynx2-8-7/src/LYUtils.c.orig	Thu Aug  2 23:24:04 2007
+++ lynx2-8-7/src/LYUtils.c	Fri Aug  3 15:59:06 2007
@@ -5903,7 +5903,7 @@ static BOOL IsOurSymlink(const char *nam
 
 	    if (cutoff != 0) {
 		HTSprintf0(&clone, "%.*s%s%s",
-			   cutoff - name,
+			   (int)(cutoff - name),
 			   name, PATHSEP_STR, buffer);
 		FREE(buffer);
 		buffer = clone;
